charts/nri-kube-events/README.md

```diff
@@ -1,57 +1,149 @@
-
-## Example
-
-Make sure you have [added the New Relic chart repository.](../../README.md#installing-charts)
-
-Then, to install this chart, run the following command:
-
-```sh
-helm install newrelic/nri-kube-events \
---set licenseKey=<enter_new_relic_license_key> \
---set cluster=my-k8s-cluster
+![Version: 2.2.2](https://img.shields.io/badge/Version-2.2.2-informational?style=flat-square) ![AppVersion: 1.8.0](https://img.shields.io/badge/AppVersion-1.8.0-informational?style=flat-square)
+
+A Helm chart to deploy the New Relic Kube Events router
+
+**Homepage:** <https://docs.newrelic.com/docs/integrations/kubernetes-integration/kubernetes-events/install-kubernetes-events-integration>
+
+## Source Code
+
+* <https://github.com/newrelic/nri-kube-events/>
+* <https://github.com/newrelic/nri-kube-events/tree/master/charts/newrelic-infrastructure>
+
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://helm-charts.newrelic.com | common-library | 1.0.2 |
+
+## Values managed globally
+
+This chart implements the [New Relic's common Helm library](https://github.com/newrelic/helm-charts/tree/master/library/common-library) which
+means that is has a seamless UX between things that are configurable across different Helm charts.
+
+Here is a list of options that you can set globally and affect all the charts in the `nri-bundle` at the same time while being able to override
+the behaviour locally only for this chart. To give more width to the __Description__ column we cut the options like `[global.]cluster`. That
+means that you can set `cluster` in the root of the values of this chart or in the global to affect all the charts from New Relic.
+
+| key | Default | Merged[<sup>1</sup>](#values-managed-globally-1) | Description |
+|-----|---------|--------------------------------------------------|-------------|
+| [global.]cluster | `""` |  | Allows to set into the integration the cluster name that is being monitored |
+| [global.]licenseKey | `""` |  | This set this license key to all the products of New Relic |
+| [global.]customSecretName | `""` |  | In case you don't want to have the license key in you values, this allows you to point to a user created secret to get the key from there |
+| [global.]customSecretLicenseKey | `""` |  | In case you don't want to have the license key in you values, this allows you to point to which secret key is the license key located |
+| [global.]podLabels | `{}` | yes | A map of labels to add to the pods deployed with this chart |
+| [global.]Labels | `{}` | yes | A map of labels to add to all the objects deployed with this chart |
+| [global.]priorityClassName | `""` |  | Allows to set pod's priorityClassName |
```

Comment:
In general I would (personally) get rid of all `Allows to set` 

However, someone else might prefer your wording

Reply:
I like it. It is shorter and more direct. Solved in [#9f9cfc0](https://github.com/newrelic/nri-kube-events/pull/112/commits/9f9cfc0e51242f18647739df110dcb9b8600a133)